service/matchers_test.go

```diff
@@ -0,0 +1,14 @@
+package service
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSvc(t *testing.T) {
+	assert := assert.New(t)
+
+	_, err := NewOsSvc("servicetst", "systemd")
```

Comment:
```service``` package refactor is in ```svc-test``` branch. Ignore this. This has been "transferred" from the old code.